scrapi/base/helpers.py

```diff
@@ -135,7 +136,10 @@ def oai_process_uris(*args):
                 provider_uris.append(found_url)
 
     try:
-        canonical_uri = (provider_uris + object_uris)[0]
+        if use_doi and len(object_uris) > 0:
+            canonical_uri = object_uris[0]
```

Comment:
A doi oftentimes does point to the sources splash page where the resource lives -- it does on dataone and others I've seen for sure